src/batches/setBatch3Dto4D.m

```diff
@@ -0,0 +1,39 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function matlabbatch = setBatch3Dto4D(optSource, volumesList)
+%
+% Short description of what the function does goes here.
+%
+% USAGE::
+%
+%   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
+%
+% :param argin1: (dimension) obligatory argument. Lorem ipsum dolor sit amet,
+%                consectetur adipiscing elit. Ut congue nec est ac lacinia.
+% :type argin1: type
+% :param argin2: optional argument and its default value. And some of the
+%               options can be shown in litteral like ``this`` or ``that``.
+% :type argin2: string
+% :param argin3: (dimension) optional argument
+%
+% :returns: - :argout1: (type) (dimension)
+%           - :argout2: (type) (dimension)
+%
+% .. todo:
+%
+%    - item 1
+%    - item 2
+
+fprintf(1, 'PREPARING: 3D to 4D conversion\n');
```

Comment:
I am creating a function to take care of the printing the batch names. Help ensuring they all have the same format. It is in a coming PR. 

But don't worry about that for now. 

Just to let you know.